sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -706,6 +707,54 @@ public virtual VerifyResult VerifyData(SignatureAlgorithm algorithm, Stream data
             }
         }
 
+        /// <summary>
+        /// Encrypts the specified key using the specified algorithm
+        /// </summary>
+        /// <param name="algorithm">The algorithm to use to encrypt the key</param>
+        /// <param name="key">The key to be encrypted</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> to cancel the operation.</param>
+        /// <returns>The encrypted key</returns>
+        byte[] IKeyEncryptionKey.WrapKey(string algorithm, byte[] key, CancellationToken cancellationToken)
+        {
+            return WrapKey(new KeyWrapAlgorithm(algorithm), key, cancellationToken).EncryptedKey;
+        }
+
+        /// <summary>
+        /// Encrypts the specified key using the specified algorithm
+        /// </summary>
+        /// <param name="algorithm">The algorithm to use to encrypt the key</param>
+        /// <param name="key">The key to be encrypted</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> to cancel the operation.</param>
+        /// <returns>The encrypted key</returns>
+        async Task<byte[]> IKeyEncryptionKey.WrapKeyAsync(string algorithm, byte[] key, CancellationToken cancellationToken)
+        {
+            return (await WrapKeyAsync(new KeyWrapAlgorithm(algorithm), key, cancellationToken).ConfigureAwait(false)).EncryptedKey;
```

Comment:
Nit: breaking apart complex statements makes it easier to debug later, i.e. make this two separate statements. IMO, makes it more readable as well.